### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by couger01**
Implementing Jenkins shared library

Update Jenkinsfile.conda

Implementing the Jenkins shared library

Removed ts_config_ocs

DSM is non-configurable so the config_repo argument should be blank.  The shared library was updated to skip the config repo checkout stage if config_repo is blank.

Update Jenkinsfile.conda

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -19,14 +19,10 @@ build:
 
 test:
   requires:
-    - pytest
```
Comment (mareuter) [inline conda/meta.yaml:22]: Just curious as to why all of this is disappearing.

File: Jenkinsfile.conda [hunk 2]
```diff
@@ -1,62 +1,2 @@
...
-    }
-    post {
-        cleanup {
-            sh """
-            docker stop ${container_name}
-            """
-        }
-    }
-}
+@Library('JenkinsShared')_
```
Comment (mareuter) [inline Jenkinsfile.conda:1]: Love how compact this it now!

**Review by mareuter — APPROVED** [review]
One small curiosity, but this looks fine to me. I don't see why the one build failed, but it looks like it tried to build every branch and tag in the repo. Is that intentional?

File: conda/meta.yaml (hunk 1 above)
Comment (couger01) [reply conda/meta.yaml:22]: So Tiago made a conda metapackage ts-conda-build which installs a number of packages in our stack. This allows us to just list that package as a dependency.

**Comment by mareuter** [comment]
Is this ready to go? Do we understand the build failure?

**Comment by couger01** [comment]
Looks like its passing now.

**Pull request merged**